packages/desktop-client/src/hooks/useAccountPreviewTransactions.ts

```diff
@@ -32,10 +32,6 @@ export function useAccountPreviewTransactions({
   const payees = usePayees();
 
   const previewTransactions = useMemo(() => {
-    if (isLoading) {
-      return [];
-    }
```

Comment:
This was causing the flicker - we reset the preview transactions array whenever we saved a transaction on the page. I think this is unnecessary.